qwandaq/src/main/java/life/genny/qwandaq/constants/FilterConst.java

```diff
@@ -0,0 +1,135 @@
+package life.genny.qwandaq.constants;
+
+public class FilterConst {
+
+    public static final String PAGINATION_NEXT="QUE_TABLE_NEXT_BTN";
+    public static final String PAGINATION_PREV="QUE_TABLE_PREVIOUS_BTN";
+    public static final String PAGINATION_INDEX="PRI_INDEX";
+    public static final String SEARCH_TEXT="PRI_SEARCH_TEXT";
+    public static final String  FILTER_LABEL = "FILTER_LABEL";
+
+    public static final String  PRI_PREFIX = "PRI_";
+    public static final String PRI_NAME="PRI_NAME";
+    public static final String PRI_CODE="PRI_CODE";
+    public static final String FIELD="FIELD_";
+    public static final String PRI_CREATED_DATE="PRI_CREATED_DATE";
+
+    /* PCM */
+    public static final String PCM_TABLE = "PCM_TABLE";
```

Comment:
PCM constants should go in the `PCM.java` class.

Reply:
Used PCM.java